opencensus/trace/internal/span_impl.cc

```diff
@@ -136,6 +136,8 @@ void SpanImpl::SetStatus(exporter::Status&& status) {
 
 void SpanImpl::End() { EndWithTime(absl::Now()); }
 
+bool SpanImpl::HasEnded() { return has_ended_; }
```

Comment:
How does this work without a MutexLock? Looks like we're missing annotalysis on Travis.

Reply:
I think technically we could get away without a lock here since it is just 1 byte being read, but let's put it in for good coding standards.